Packages/com.unity.inputsystem/InputSystem/Editor/InputEditorAnalytics.cs

```diff
@@ -0,0 +1,343 @@
+#if UNITY_EDITOR
+
+using System;
+using UnityEngine.InputSystem.LowLevel;
+
+namespace UnityEngine.InputSystem.Editor
+{
+    internal static class InputEditorAnalytics
+    {
+        [Serializable]
+        public struct InputActionsEditorSessionData : UnityEngine.InputSystem.InputAnalytics.IInputAnalyticData
+        {
+            /// <summary>
+            /// Represents an editor type.
+            /// </summary>
+            /// <remarks>
+            /// This may be added to in the future but items may never be removed.
+            /// </remarks>
+            public enum Kind
+            {
+                Invalid = 0,
+                FreeFloatingEditorWindow = 1,
```

Comment:
does that describe the editor that is opened from a custom created asset? It sounds a bit like this is a window that is not attached to any area in the editor but rather lays in another new window.

Reply:
Yes, it implies that the asset is being edited in a window (not reflecting at all how it was opened).
I have renamed it to EditorWindow only